core/esmf-aspect-model-validator/src/main/java/org/eclipse/esmf/aspectmodel/shacl/ShaclValidator.java

```diff
@@ -128,7 +125,8 @@ public List<Violation> validateModel( final VersionedModel model ) {
    private Map<Resource, List<Shape.Node>> findSparqlTargets( final Model model ) {
       final Map<Resource, List<Shape.Node>> resourceShapes = new HashMap<>();
       for ( final Shape.Node shape : targetSparqlShapes() ) {
-         final List<Resource> shapeTargets = querySparqlTargets( model, shape.attributes().targetSparql().get() );
+         final List<Resource> shapeTargets = querySparqlTargets( model, shape.attributes().targetSparql().orElseThrow( () ->
+               new RuntimeException( "SPARQL node shape is missing a target SPARQL expression" ) ) );
```

Comment:
I would not throw a RuntimeException. Maybe a own exception or Illegal exception?